### PR TITLE
THEMES-996: Add blocks to commerce

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Added the following blocks/Chanins to Commerce:

Blocks:
* Lead Art - Arc Block
* Headline - Arc Block
* Subheadline - Arc Block
* Date - Arc Block
Chains:
* Article Body - Arc Block

## Jira Ticket

- [THEMES-996](https://arcpublishing.atlassian.net/browse/THEMES-996)

## Acceptance Criteria

1. As a user in Themes Settings, I can select the following blocks/chains to add to my bundle:
  a. Current Blocks/Layouts (these are currently available in Arc Commerce):
  b. New Blocks:
    * Lead Art - Arc Block
    * Headline - Arc Block
    * Subheadline - Arc Block
    * Date - Arc Block
  c. New Chains:
    * Article Body - Arc Block
2. Copy/paste from News theme and evaluate/make followup ticket for any additional major styling updates

## Test Steps
1. Checkout this branch `git checkout themes-996`
2. Run storybook `npm run storybook`
3. Check  http://localhost:60001/ for new blocks

## Dependencies or Side Effects

https://github.com/WPMedia/arc-themes-manifests/pull/165
https://github.com/WPMedia/arc-themes-feature-pack/pull/385

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-996]: https://arcpublishing.atlassian.net/browse/THEMES-996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .storybook/themes/commerce.scss [hunk 1]
```diff
@@ -1215,8 +1567,11 @@
 
 @use "../../blocks/alert-bar-block";
 @use "../../blocks/category-carousel-block";
+@use "../../blocks/date-block";
 @use "../../blocks/default-output-block";
+@use "../../blocks/header-nav-chain-block";
 @use "../../blocks/hero-block";
+@use "../../blocks/large-promo-block";
```
Comment (malavikakoppula) [inline .storybook/themes/commerce.scss]: @nschubach , As large-promo styles are not added, can we remove this?

Comment (nschubach) [reply .storybook/themes/commerce.scss]: Oh, that's an interesting call out.  I believe I copied the use from the storybook and updated it, need to verify that is not wrong too...

Comment (nschubach) [reply .storybook/themes/commerce.scss]: @malavikakoppula the used blocks have been corrected.

**Review by malavikakoppula — APPROVED** [review]
LGTM

**Pull request merged**